src/import/ImportAUP.cpp

```diff
@@ -911,43 +900,26 @@ bool AUPImportFileHandle::HandleTags(XMLTagHandler *&handler)
    wxString v;
 
    // Support for legacy tags
-   while(*mAttrs)
+   for (auto pair : mAttrs)
    {
-      const wxChar *attr = *mAttrs++;
-      const wxChar *value = *mAttrs++;
+      auto attr = pair.first;
+      auto value = pair.second;
 
-      if (!value)
-      {
-         break;
-      }
-      
-      // Ignore empty tags
-      if (!*value)
```

Comment:
The skipping of empty tags was not preserved

Reply:
Fixed!